src/Core/Prefab/GlobeView.js

```diff
@@ -8,18 +8,80 @@ import RendererConstant from '../../Renderer/RendererConstant';
 import GlobeControls from '../../Renderer/ThreeExtended/GlobeControls';
 import { unpack1K } from '../../Renderer/MatteIdsMaterial';
 
-import { GeometryLayer } from '../Layer/Layer';
+import { GeometryLayer, ImageryLayers } from '../Layer/Layer';
 
 import Atmosphere from './Globe/Atmosphere';
+import CoordStars from '../Geographic/CoordStars';
 import Clouds from './Globe/Clouds';
 
 import { C, ellipsoidSizes } from '../Geographic/Coordinates';
 import { processTiledGeometryNode, initTiledGeometryLayer } from '../../Process/TiledNodeProcessing';
 import { updateLayeredMaterialNodeImagery, updateLayeredMaterialNodeElevation } from '../../Process/LayeredMaterialNodeProcessing';
 import { globeCulling, preGlobeUpdate, globeSubdivisionControl, globeSchemeTileWMTS, globeSchemeTile1 } from '../../Process/GlobeTileProcessing';
 import BuilderEllipsoidTile from './Globe/BuilderEllipsoidTile';
+import Fetcher from '../../Core/Scheduler/Providers/Fetcher';
+
+export const GLOBE_VIEW_EVENTS = {
+    GLOBE_INITIALIZED: 'initialized',
+    LAYER_ADDED: 'layer-added',
+    LAYER_REMOVED: 'layer-removed',
+    LAYER_CHANGED: 'layer-changed',
+    LAYER_VISIBILITY_CHANGED: 'layer-visibility-changed',
+    LAYER_OPACITY_CHANGED: 'layer-opacity-changed',
+    LAYER_INDEX_CHANGED: 'layer-index-changed',
+};
 
-
+/**
+ * The intellectual property rights
+ * @typedef {Object} Attribution
+ * @property {string} name
+ * @property {string} url
+ */
+
+/**
+ * Options to wms protocol
+ * @typedef {Object} OptionsWms
+ * @property {Attribution} attribution The intellectual property rights for the layer
+ * @property {string} name
+ * @property {string} mimetype
+ */
+
+/**
+ * Options to wtms protocol
+ * @typedef {Object} OptionsWmts
+ * @property {Attribution} attribution The intellectual property rights for the layer
+ * @property {string} name
+ * @property {string} mimetype
+ * @property {string} tileMatrixSet
+ * @property {Array.<Object>} tileMatrixSetLimits The limits for the tile matrix set
+ * @property {number} tileMatrixSetLimits.minTileRow Minimum row for tiles at the level
+ * @property {number} tileMatrixSetLimits.maxTileRow Maximum row for tiles at the level
+ * @property {number} tileMatrixSetLimits.minTileCol Minimum col for tiles at the level
+ * @property {number} tileMatrixSetLimits.maxTileCol Maximum col for tiles at the level
+ * @property {Object} [zoom]
+ * @property {Object} [zoom.min] layer's zoom minimum
+ * @property {Object} [zoom.max] layer's zoom maximum
+ */
+
```

Comment:
Oh I get it. You're documenting `addLayer`.
Could you move this above `addLayer` please?